Detectors/TPC/calibration/include/TPCCalibration/IDCAverageGroup.h

```diff
@@ -0,0 +1,141 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file IDCAverageGroup.h
+/// \brief class for averaging/grouping and storing of IDCs
+/// \author Matthias Kleiner <mkleiner@ikf.uni-frankfurt.de>
+
+#ifndef ALICEO2_IDCAVERAGEGROUP_H_
+#define ALICEO2_IDCAVERAGEGROUP_H_
+
+#include <vector>
+#include "TPCCalibration/IDCGroup.h"
+#include "TPCBase/Mapper.h"
+
+#if (defined(WITH_OPENMP) || defined(_OPENMP)) && !defined(__CLING__)
+#include <omp.h>
+#endif
+
+namespace o2
+{
+namespace tpc
+{
```

Comment:
I think in all places this can be reduced to
```suggestion
namespace o2::tpc
{
```